core/src/testUtils/groovy/nu/yona/server/test/User.groovy

```diff
@@ -14,6 +14,7 @@ class User
 	final String firstName
 	final String lastName
 	final String mobileNumber
+	final String creationTime
```

Comment:
Please declare this as ``ZonedDateTime creationTime``. See ``Goal.groovy`` for similar code.